app/Swagger/schemas.php

```diff
@@ -348,4 +349,33 @@ class RSVPUpdateRequestSchema_{
 
     ]
 )]
-class RSVPAdminAddRequestSchema {}
\ No newline at end of file
+class RSVPAdminAddRequestSchema {}
+
+#[OA\Schema(
```

Comment:
@matiasperrone-exo we need to start to break down this schema file on several files ( like one file responsability
* marketplace
* rsvp
* general 
and so